rcsb/app/file/uploadRequest.py

```diff
@@ -86,10 +86,13 @@ async def upload(
     hashDigest: str = Form(None),
     # save file parameters
     filePath: str = Form(...),
+    fileSize: int = Form(None),
```

Comment:
It's for a web upload. They may prefer to use file size comparison instead of hash comparison. The default None makes optional since they could send a hash instead.